ci/cloudbuild/notifiers/bigquery/bigquery.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: cloud-build-notifiers/v1
+kind: BigQueryNotifier
+metadata:
+  name: google-cloud-cpp-bigquery-notifier
+spec:
+  notification:
+    filter: build.substitutions["REPO_NAME"] == "google-cloud-cpp"
+    delivery:
+      table: projects/936212892354/datasets/cloudbuild/tables/google_cloud_cpp
```

Comment:
nit: can you use the project id (`cloud-cpp-testing-resources`) instead of the project number?

Reply:
I thought it had to be the numeric ID, but I think I was mistaken. Using `cloud-cpp-testing-resources` worked too. Changed.